systemd/docker.socket

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Docker Socket for the API
+PartOf=docker.service
+
+[Socket]
+ListenStream=/var/run/docker.sock
```

Comment:
Do we also have to modify the main unit file to have `depends on / after docker.socket` and `-H fd://` in the `ExecStart` ?

Reply:
Yup that's part of the `00-socket-activation.conf` drop in file now